pkg/capacity/capacity.go

```diff
@@ -358,8 +385,11 @@ func (c *Controller) addWorkItem(segment *topology.Segment, sc *storagev1.Storag
 		storageClassName: sc.Name,
 	}
 	// Ensure that we have an entry for it...
-	capacity := c.capacities[item]
-	c.capacities[item] = capacity
+	_, found := c.capacities[item]
```

Comment:
This change here alone is not functionally identical: the new line only checks for the capacity without adding it.

But directly underneath it we set the entry to nil if not found, and together with that it is functionally equivalent because `capacity := c.capacities[item]` will also be nil when not found.